pytket/extensions/cutensornet/mps/mps.py

```diff
@@ -546,11 +583,13 @@ def measure(self, qubits: set[Qubit]) -> dict[Qubit, int]:
             # Since the MPS is in canonical form, this corresponds to the probability
             # if we were to take all of the other tensors into account.
             prob = cq.contract(
-                "lrp,lrP,p,P->",  # No open bonds remain; this is just a scalar
+                "lrp,p,lrP,P->",  # No open bonds remain; this is just a scalar

```

Comment:
Was that a bug?

Reply:
Nope, I also changed the order of the operators in the lines below, so these had to be changed accordingly to match. The reason I did this was just so that the contraction path was easier to specify.

Reply:
Ah cool, understood, thanks!